primitives/allocator/src/freeing_bump.rs

```diff
@@ -86,7 +86,7 @@ macro_rules! trace {
 // This number corresponds to the number of powers between the minimum possible allocation and
 // maximum possible allocation, or: 2^3...2^24 (both ends inclusive, hence 22).
 const N_ORDERS: usize = 22;
-const MAX_POSSIBLE_ALLOCATION: u32 = 16777216; // 2^24 bytes, 16 MiB
+const MAX_POSSIBLE_ALLOCATION: u32 = 33554432; // 2^25 bytes, 32 MiB
```

Comment:
You caught me on the first minute of awakening and I spoke too soon : ) 

We need to bump not only this constant but also the `N_ORDER` and update all associated places where we assume 22 orders and the places where we refer to the 24th power of 2.

Reply:
glad you caught the issue! will fix.